documentation/asciidoc/computers/configuration/boot_folder.adoc

```diff
@@ -85,12 +85,12 @@ Various xref:linux_kernel.adoc#kernel[kernel] image files that correspond to Ras
 
 | `kernel8.img`
 | BCM2837, BCM2711, BCM2712
-| Pi Zero 2 W, Pi 2 (later revisions), Pi 3, CM3, Pi 3+, CM3+, Pi 4, CM4, CM4S, Pi 400, CM5, Pi 5, Pi 500
+| Pi Zero 2 W, Pi 2 (later revisions), Pi 3, CM3, Pi 3+, CM3+, Pi 4, CM4, CM4S, Pi 400, CM5, Pi 5, Pi 500, Pi 500+
 | xref:config_txt.adoc#boot-options[64-bit kernel]. Earlier revisions of Raspberry Pi 2 (with BCM2836) do not support 64-bit kernels.
 
 | `kernel_2712.img`
 | BCM2712
-| Pi 5, CM5, Pi 500
+| Pi 5, CM5, Pi 500, Pi 500+
```

Comment:
Also - Pi4 / Pi5 family rather than listing every possible model

Reply:
In general, I think listing every possible product tends to result in accidental omissions and implies that there it's specific to the board rather than the chip.
As someone building the kernel you are probably more interested in the architecture that it's supporting. So kernel_2712 is any 2712 device.

Reply:
Sounds good. Something copy-and-pasteable :)